master/buildbot/test/unit/test_steps_http.py

```diff
@@ -99,15 +99,19 @@ def test_404(self):
 
     def test_POST(self):
         url = self.getURL("POST")
-        content = "\n<html>\n" \
-                  "  <head><title>405 - Method Not Allowed</title></head>\n" \
-                  "  <body>\n    <h1>Method Not Allowed</h1>\n    <p>Your browser " \
-                  "approached me (at /POST) with the method \"POST\".  I only allow the " \
-                  "methods HEAD, GET here.</p>\n  </body>\n</html>\n"
+        #
```

Comment:
in this, case, please just drop the comment, but please don't leave commented out dead code.